_dasldemos/fh-cbio-intro.md

```diff
@@ -0,0 +1,247 @@
+        ```
+
+7. Our Airflow automation scripts will take care of the rest of the upload process from there and send you an email notification with details about the outcome. If upload was unsuccessful, the email will contain a detailed HTML file called a "validation report" that identifies which parts of your study are causing the issue and need updating. *Note:* If you do not receive an email notification indicating the success/failure of your study upload within an hour, reach out to the [cBioPortal Team](mailto:tfirman@fredhutch.org) for help identifying the issue.
+
+8. Go have fun and explore your data [on the Fred Hutch instance of cBioPortal!](https://cbioportal.fredhutch.org/)
+
+9. If you would like to update your study data or add more subjects to the cohort in the future, simply update your study data locally and reupload it to `fh-dasl-cbio`. Any subsequent uploads of the same study will overwrite the previous study data.
+
+## How do I prepare my data for upload into cBioportal?
+
+### Before you begin
+- Start by creating a folder to store all of your relevant study files. **The name of the folder must be the exact study ID that was provided to you by the cBioPortal team.** If any other name is provided, the automation steps will refuse upload since the altered study ID would not be on the list of approved values. *Note: the `cancer_study_identifier` value in your meta files should also be this same study ID.*
+- Once your study files are close to complete, you can test out your file formatting using one of two validation methods:
+    1. If you've received official approval for upload, try directly uploading it into the `fh-dasl-cbio` bucket. As mentioned above, the automated upload process will send you a validation report via email outlining any issues that may have popped up.
+    2. You could also validate your study folder by [launching a local instance](https://github.com/FredHutch/tg-cbioportal-data/03_launch_local) of cBioPortal and troubleshooting the errors (if any) in your study data.
+
+### Required/Optional Study Files
+There are a few files that are required while all other files are optional. Below is an overview of the required files and some optional files *Note: Version 6 of cBioportal currently also requires in the least 1 non-clinical file to be uploaded as well. See instructions below on where to find a dummy table that you can modify in case you are only uploading clinical data.*
+
+| Type                                        | Requirement                                        | Filename example                              | Required Format                     | Purpose                                                              | Detailed Instructions                                                                              | Example                                                                                               |
```

Comment:
TO me it would make more sense to have all the required files be at the top and optional below, but I'm not fully aware of the context so maybe it makes sense as is

Reply:
Agreed, resorted the table to start with required files and end with optional files.